src/meme-upload/local.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs')
+const path = require('path')
+const shortid = require('shortid')
+
+function add(stream) {
+  const fileName = shortid.generate() + '.mp3'
+
+  const filePath = path.join(process.env.MEME_DIR, fileName)
+  stream.pipe(fs.createWriteStream(filePath))
+
+  const url = `/memes/${fileName}`
```

Comment:
We should unify the url field between local and spaces (i.e. the url in Mongo for local storage should be the fully-qualified name), but I'm not sure how to do that easily.

Reply:
Yeah I was thinking about that and totally forgot to change that.